tcpdf.php

```diff
@@ -13670,11 +13674,31 @@ public function setTimeStamp($tsa_host='', $tsa_username='', $tsa_password='', $
 	 * @author Richard Stockinger
 	 * @since 6.0.090 (2014-06-16)
 	 */
+	
+	/**
+	 * Applying TSA for a timestamp.
+	 * @param string $signature Digital signature as hex string
+	 * @return hex string Timestamped digital signature
+	 * @protected
+	 * @author M Hida
+	 * @since 6.6.2 (2023-05-25)
+	 */
```

Comment:
This documentation is misleading since the function already existed in prior versions
could you update this to reflect that it already existed ?

Reply:
probably remove the previous comment, right?

Reply:
thanks, that's better